index.bs

```diff
@@ -447,6 +447,8 @@ When this method is invoked, the user agent MUST execute the following algorithm
     unless the {{ScopedCredentialOptions/attachment}} member of {{options}} is <a>present</a>. In that case, let |attachment|
     be {{ScopedCredentialOptions/attachment}}, and add the authenticator to |currentlyAvailableAuthenticators| if its attachment
     modality matches |attachment|.
+    - If no authenticator is available, reject |promise| with a <a>DOMException</a> whose name is "NotFoundError", and 
```

Comment:
I'd make this a separate step after this one:
> If |currentlyAvailableAuthenticators| is empty, [=reject=] |promise| with a "{{NotFoundError}}" {{DOMException}} and terminate this algorithm.

(See the example in https://heycam.github.io/webidl/#idl-exceptions for that exception creation wording.)